docs/learn/specific-css.mdx

```diff
@@ -1,29 +1,27 @@
 ---
-slug: /specific-css
-title: Argos-specific CSS
+slug: /injecting-css
 sidebar_position: 3
 ---
 
-# Argos-specific CSS
+# Injecting CSS
 
-Understand the method for using Argos-focused CSS to exclude certain sections from appearing in your screenots.
+Argos integrations offer the possibility to inject custom CSS applied during the screenshot process to improve your screenshots or exclude some details.
 
-In addition to [Argos built-in helpers](https://argos-ci.com/docs/argos-helpers#helpers), Argos integrations offer the possibility to inject custom CSS applied during the screenshot process. It can be helpful for complex use cases that produce flaky visual diffs.
+## Usage
```

Comment:
There is two methods you can't name one "Usage"

Reply:
I didn't understand the second section of the page. I wanted to ask you before updating it.

Reply:
Leave it like that then, you can rename it to "Use Argos class"